label.py

```diff
@@ -1,17 +1,20 @@
-import json
+import csv
 from PIL import ImageTk, Image
 from tkinter import Tk, Label
 
 import utils
 
 
 try:
```

Comment:
Could you move the code to read labels into a function in utils.py?

Reply:
Ok! I will add it to utils.